easybuild/tools/py2vs3/py2.py

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2019-2019 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Functionality to facilitate keeping code compatible with Python 2 & Python 3.
+
+Implementations for Python 2.
+
+:author: Kenneth Hoste (Ghent University)
+"""
+# these are not used here, but imported from here in other places
+import ConfigParser as configparser  # noqa
+from string import letters as ascii_letters  # noqa
+from StringIO import StringIO  # noqa
+from urllib2 import HTTPSHandler, Request, build_opener  # noqa
+
+# string type that can be used in 'isinstance' calls
+string_type = basestring
+
+
+def raise_with_traceback(exception_class, message, traceback):
+    """Raise exception of specified class with given message and traceback."""
+    raise exception_class, message, traceback
```

Comment:
This one can be ignored, Hound is only complaining here because this is Python 2 syntax (cfr. https://stackoverflow.com/questions/47427916/how-to-config-hound-ci-to-support-python2-7)

Reply:
And we cannot tell it to ignore this line?

Reply:
I don't think so...

It will only make a comment when the code is touched, so it's not an issue imho.